inc/namespace.php

```diff
@@ -1717,3 +1725,69 @@ function split_large_ep_request( $request, array $query, array $args, int $failu
 
 	return $request;
 }
+
+/**
+ * Handle request forwarding to ES.
+ */
+function handle_autosuggest_endpoint() {
```

Comment:
@roborourke could this actually just be a regular REST API call?

Reply:
That'd need to be handled upstream but presumably for elasticpress.io interop the answer is no. The plugin's JS is where the request is made so would need to modify that.